Modules/Core/Common/test/itkFixedArrayGTest.cxx

```diff
@@ -324,3 +325,22 @@ TEST(FixedArray, StdMemberFunctionsWork)
   d3arr.data()[0] = 10;
   EXPECT_EQ(cdata[0], 10);
 }
+
+
+// Tests that each element of a value-initialized FixedArray is itself value-initialized. By definition,
+// "value-initialization" is performed when an object is constructed with an empty {} initializer.
+TEST(FixedArray, ValueInitialized)
+{
+  const auto expectEachElementValueInitialized = [](const auto & fixedArray) {
+    for (const auto & element : fixedArray)
+    {
+      using ValueType = std::remove_reference_t<decltype(element)>;
+
+      EXPECT_TRUE(element == ValueType{});
```

Comment:
I would be more explicit here, and compare element to new lambda parameter `expectedValue`.